src/shared/components/post-listing.tsx

```diff
@@ -169,6 +169,7 @@ export class PostListing extends Component<PostListingProps, PostListingState> {
       <PictrsImage
         src={src}
         thumbnail
+        alt=""
```

Comment:
If its an optional prop, you don't need to include it here.

Reply:
Will remove